### PR TITLE
Add simple docs for base and ceq

### PR DESCRIPTION
**Pull request by david-christiansen**

### CODE REVIEW

**Comment by jozefg** [comment]
Should we attach some intuition for what `ceq` "means" practically? Even if it was just

> Two terms are computationally equivalent if they both diverge or run to equivalent results. Computational equivalence is a congruence so one can also prove that two terms are computationally equal if their subterms are computational equal.

Though perhaps a more in depth explanation may be relegated to somewhere else?

Otherwise :+1: from me :)

**Comment by david-christiansen** [comment]
Good call. Updated.

**Comment by david-christiansen** [comment]
I think some more undocumented operators have also crept in - gotta get completion and doc lookup working one of these days :-)

File: README.md [hunk 1]
```diff
@@ -143,6 +143,10 @@ Operator ex_uni : (0;1).
 Note that `P` is applied to `x` and `y` using `so_apply` rather than
 `ap`, which is reserved for function application.
 
+#### Computational Equivalence
+
+The elements of `base()` are all closed terms. Their equality is `ceq(0,0)`, which denotes [Howe's computational equivalence](http://www.nuprl.org/KB/show.php?ShowPub=Howe89). Two terms are computationally equivalent if they both diverge or if they run to equivalent results. Computational equivalence is a congruence, which means that one can also prove that two terms are computationally equivalent if their subterms are computationally equivalent.
```
Comment (jonsterling) [inline README.md]: Can we change this to `ceq(0;0)` (semicolon)?

**Comment by david-christiansen** [comment]
Sorry, that always messes me up.

**Comment by jonsterling** [comment]
thanks very much!

**Pull request merged**